test/e2e/README.md

```diff
@@ -48,9 +44,9 @@ export POSITRON_PY_ALT_VER_SEL='3.13.0 (Pyenv)'
 export POSITRON_R_ALT_VER_SEL='4.4.2'
 ```
 
-_Note: The environment variables with `ALT` in them are for the alternate interpreters which are used in fewer tests and might not be immediately needed. If you are using Pyenv for your alternate Python interpreter, please add ` (Pyenv)` to the variable value._
+Make sure you have the selected R and Python version installed that you are using for the environment variables.
 
-_Note: If you forgot to set the environment variables before running the tests, you'll need to restart your editor or shell session for the environment variables to be loaded in._
+_Note: The environment variables with `ALT` in them are for the alternate interpreters which are used in fewer tests and might not be immediately needed. If you are using Pyenv for your alternate Python interpreter, please add ` (Pyenv)` to the variable value._
```

Comment:
I am not sure its true that the ALT interpreters are not immediately needed. I think one will encounter startup errors if they are not set.